Automattic/Gravatar-SDK-iOS
# Adding inline docs to base profile view

Closes #

### Description

Adding inline docs to base profile view.

### Testing Steps

- Build Docs
- Search for `BaseProfileView`.
- Check that all reads good.
